### PR TITLE
Mageaasim

### LINKED ISSUE #30589: Incorrect currency symbol for surinamese dollar
### Preconditions (*)

Magento 2.4-develop
### Steps to reproduce (*)

1. set base currency to Suriname Dollar

### Expected result (*)

1. prices on the catalog and the frontend to be prepended with the SRD symbol
image

### Actual result (*)

1. prices on the catalog and the frontend are prepended with the $ symbol
2020-10-21-102115_410x319_scrot

---
Please provide [Severity](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#backlog) assessment for the Issue as Reporter. This information will help during Confirmation and Issue triage processes.

- [x] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [ ] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [x] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

### PR DESCRIPTION
**Pull request by aasim110**
### Fixed Issues (if relevant)

1. Fixes magento/magento2#30589

### Manual testing scenarios (*)

1. Set the Surinamese (SRD) currency from the admin
2. Currency symbol with the products on the frontend and admin area is working fine as expected
3. Unit Tests
4. Sample Data Tests CE

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by aasim110** [comment]
@magento give me 2.4-develop instance

**Comment by aasim110** [comment]
@magento give me test instance

**Comment by aasim110** [comment]
@magento run all tests

**Comment by aasim110** [comment]
@magento run Unit Tests

File: 485c714f96498a91d7346062697997c66df4ad0a [hunk 1]
```diff
@@ -0,0 +1,3630 @@
+[33mcommit 485c714f96498a91d7346062697997c66df4ad0a[m[33m ([m[1;36mHEAD -> [m[1;32mmageaasim[m[33m, [m[1;31mupstream/2.4-develop[m[33m, [m[1;31morigin/HEAD[m[33m, [m[1;31morigin/2.4-develop[m[33m, [m[1;32m2.4-develop[m[33m)[m
```
Comment (gabrieldagama) [inline 485c714f96498a91d7346062697997c66df4ad0a]: Please delete this file.

File: vendor/magento/zendframework1/library/Zend/Locale/Data/nl_SR.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (gabrieldagama) [inline vendor/magento/zendframework1/library/Zend/Locale/Data/nl_SR.xml:1]: We should not push changes on the vendor folder, that is handled by composer.

File: vendor/magento/zendframework1/library/Zend/Locale/Data/root.xml [hunk 3]
```diff
@@ -0,0 +1,3771 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (gabrieldagama) [inline vendor/magento/zendframework1/library/Zend/Locale/Data/root.xml:1]: We should not push changes on the vendor folder, that is handled by composer.

**Review by gabrieldagama — CHANGES_REQUESTED** [review]
Hi @aasim110, thanks for your pull request, please have a look at my comments below.

File: vendor/magento/zendframework1/library/Zend/Locale/Data/nl_SR.xml (hunk 2 above)
Comment (aasim110) [reply vendor/magento/zendframework1/library/Zend/Locale/Data/nl_SR.xml:1]: Then how can I update the changes related to the vendor?

File: vendor/magento/zendframework1/library/Zend/Locale/Data/root.xml (hunk 3 above)
Comment (aasim110) [reply vendor/magento/zendframework1/library/Zend/Locale/Data/root.xml:1]: Then how can I update the changes related to the vendor?

File: vendor/magento/zendframework1/library/Zend/Locale/Data/nl_SR.xml (hunk 2 above)
Comment (hostep) [reply vendor/magento/zendframework1/library/Zend/Locale/Data/nl_SR.xml:1]: @aasim110: the zf1 repo is located here: https://github.com/magento/zf1/
You can try to send in a PR over there.
But the response times over there are much worse than they are on the magento2 repo (@gabrieldagama: hint hint, maybe somebody could have a look at the open PR's there)

**Comment by gabrieldagama** [comment]
Hi @aasim110, as mentioned by @hostep, please target such changes to the https://github.com/magento/zf1/ repository.

**Pull request closed without merging**